opencolorio_config_aces/config/reference/generate/config.py

```diff
@@ -968,6 +1013,87 @@ def style_to_display_colorspace(
         return colorspace
 
 
+def config_basename_aces(
+    config_mapping_file_path=PATH_TRANSFORMS_MAPPING_FILE_REFERENCE,
+):
+    """
+    Generate the *aces-dev* reference implementation *OpenColorIO* Config
+    basename.
+
+    Parameters
+    ----------
+    config_mapping_file_path : str, optional
+        Path to the *CSV* mapping file.
+
+    Returns
+    -------
+    str
+        *aces-dev* reference implementation *OpenColorIO* Config basename.
+    """
+
+    return (
+        f"reference-config_aces-{version_aces_dev()}_"
+        f"ocio-{ocio.__version__}_"
+        f"colorspaces-{version_config_mapping_file(config_mapping_file_path)}"
+        f".ocio"
+    )
+
+
+def config_name_aces(
+    config_mapping_file_path=PATH_TRANSFORMS_MAPPING_FILE_REFERENCE,
+):
+    """
+    Generate the *aces-dev* reference implementation *OpenColorIO* Config name.
+
+    Parameters
+    ----------
+    config_mapping_file_path : str, optional
+        Path to the *CSV* mapping file.
+
+    Returns
+    -------
+    str
+        *aces-dev* reference implementation *OpenColorIO* Config name.
+    """
+
+    return (
+        f"Academy Color Encoding System - Reference Config "
+        f"[ACES {version_aces_dev()}] "
+        f"[OCIO {ocio.__version__}] "
+        f"[COLORSPACES {version_config_mapping_file(config_mapping_file_path)}]"
+    )
+
+
+def config_description_aces():
+    """
+    Generate the *aces-dev* reference implementation *OpenColorIO* Config
+    description.
+
+    Returns
+    -------
+    str
+        *aces-dev* reference implementation *OpenColorIO* Config description.
+    """
+
+    header = (
+        f'The "Academy Color Encoding System" (ACES {version_aces_dev()}) '
+        f'"Reference Config"'
+    )
+    underline = "-" * len(header)
+    description = (
+        'This "OpenColorIO" config is a strict and quasi-analytical '
+        'implementation of "aces-dev" and is designed as a reference for '
+        "software developers. It is not a replacement for the previous "
+        '"ACES" configs nor the "ACES Studio Config".'
+    )
```

Comment:
Some more detail may be useful.  Suggest changing "as a reference for software developers" to "as a reference to validate the implementation of the ampas/aces-dev GitHub transforms in OpenColorIO".

In addition, perhaps add to the end of the last sentence: "... since this config only contains the transforms from aces-dev and not the additional transforms that are often needed in a VFX setting."

Reply:
Ditto!